src/SFA.DAS.EmployerAccounts.Web/Authorization/AuthorizationContextProvider.cs

```diff
@@ -15,7 +16,8 @@ public class AuthorizationContextProvider : IAuthorizationContextProvider
         private readonly IHashingService _hashingService;
         private readonly IAuthenticationService _authenticationService;
 
-        public AuthorizationContextProvider(HttpContextBase httpContext, IHashingService hashingService, IAuthenticationService authenticationService)
+        public AuthorizationContextProvider(HttpContextBase httpContext, IHashingService hashingService,
```

Comment:
*style* Either have all params on the same line, or if too long then have each on it's own line